Nodejs/Product/Nodejs/Project/NodejsFileNode.cs

```diff
@@ -39,7 +39,9 @@ public NodejsFileNode(NodejsProjectNode root, ProjectElement e)
                 return;
             }
 
-            if (Url.EndsWith(NodejsConstants.TypeScriptDeclarationExtension, StringComparison.OrdinalIgnoreCase) && Url.IndexOf(@"\typings\", StringComparison.OrdinalIgnoreCase) >= 0) {
+            if (Url.EndsWith(NodejsConstants.TypeScriptDeclarationExtension, StringComparison.OrdinalIgnoreCase)
+              && Url.IndexOf(@"\typings\", StringComparison.OrdinalIgnoreCase) >= 0
+              && Url.IndexOf(@"\node_modules\", StringComparison.OrdinalIgnoreCase) < 0) {
```

Comment:
It may be better to check if the url starts with `$PROJECT_ROOT/typings` instead of this approach.